pulseaudio-control.bash

```diff
@@ -211,9 +211,14 @@ function nextSink() {
         pacmd move-sink-input "$i" "$newSink"
     done
 
+    notify="notify-send"
```

Comment:
This part should go inside the ` if [ $NOTIFICATIONS = "yes" ]; then`. I would personally add the fallback `notify` value in an `else` block.

Also, why did you choose that exact ID? Any particular reason, or is a random number OK? Not sure how this random ID should be obtained.

Reply:
Thanks for pointing that out. That's very reasonable. I'll move it to the `if` block.

The ID is completely random. I've noticed that `dunstify` uses small, incremental values initially (two-digits) and it seems to reset after reboot. So as long as we have a large enough number, we're unlikely to have any conflicts. However, even if a conflict happens, it will only happen for notification 201839192, and that is only once. Every subsequent notification won't be affected. From my experience working with other scripts, this has never been an issue.

Reply:
Fair enough. For what I've seen everyone seems to pick a random ID, too: https://github.com/search?p=1&q=dunstify+--replace&type=Code